### PR TITLE
fix: apply localization, escape special HTML chars in license strings, CSS changes

### PR DESCRIPTION
**Pull request by SteelWagstaff**
Book cards print the full name of each book's copyright license. Sometimes, these strings contain apostrophes and other special HTML characters which weren't being unescaped. This PR fixes that problem. It also applies minor CSS improvements to address the display of list items in book cards, and updates the translations for French. Addresses #127, #254, #292 and #293

### CODE REVIEW

File: assets/css/book-card.css [hunk 1]
```diff
@@ -28,10 +28,19 @@
   color: var(--primary-dark);
 }
 
-.book-cards .book-card .book-info p {
+.book-cards .book-card .book-info * {
```
Comment (SteelWagstaff) [inline assets/css/book-card.css]: Apply desired font size to all elements in the book card book info block

File: assets/css/book-card.css [hunk 2]
```diff
@@ -28,10 +28,19 @@
...
   @apply text-base m-0;
 }
 
+.book-cards .book-card .book-info ol {
+	@apply list-decimal list-inside;
```
Comment (SteelWagstaff) [inline assets/css/book-card.css:36]: Apply desired list styling to ordered and unordered lists.

File: assets/css/book-card.css [hunk 3]
```diff
@@ -57,7 +66,7 @@
   @apply mb-2;
 }
 
-.book-cards .book-card .book-info .book-description.line-clamp *:not(a):not(em):not(del):not(span):not(strong):not(u) {
+.book-cards .book-card .book-info .book-description.line-clamp *:not(a):not(em):not(del):not(span):not(strong):not(u):not(li) {
```
Comment (SteelWagstaff) [inline assets/css/book-card.css:69]: Show the list styling for clamped items

**Review by arzola — APPROVED** [review]
LGTM, tested and working as expected

**Pull request merged**